0-bootstrap/README.md

```diff
@@ -1,59 +1,152 @@
 # 0-bootstrap
 
+This repo is part of a multi-part guide that shows how to configure and deploy
+the example.com reference architecture described in
+[Google Cloud security foundations guide](https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf)
+(PDF). The following table lists the parts of the guide.
+
+<table>
+<tbody>
+<tr>
+<td>0-bootstrap (this file)</td>
+<td>Bootstraps a Google Cloud organization, creating all the required resources
+and permissions to start using the Cloud Foundation Toolkit (CFT). This
+step also configures a CI/CD pipeline for foundations code in subsequent
```

Comment:
At this point we used `CI/CD` and in the other parts of this document we use `CICD` .
Which one should be the one to be the used?

Reply:
I think we should converge on using CI/CD but both terms are correct https://en.wikipedia.org/wiki/CI/CD